exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
+let greeting = "My name is";
+let myName = "Boshra";
+let message = greeting + " " + myName;
```

Comment:
This looks good. How else do you think you can improve the RHS ( Right Hand Side ) on line 4 though? You're currently using `+` to concatenate the strings together - but what other techniques can you use for forming strings using variables. Try checking out [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#string_interpolation) and see if you can refactor your code :)